cmd/crowdsec-cli/notifications.go

```diff
@@ -135,6 +143,81 @@ func NewNotificationsCmd() *cobra.Command {
 		},
 	}
 	cmdNotifications.AddCommand(cmdNotificationsInspect)
+	var cmdNotificationsReinject = &cobra.Command{
+		Use:               "reinject",
+		Short:             "reinject alerts into notifications system",
+		Long:              `Reinject alerts into notifications system`,
+		Example:           `cscli notifications reinject <alert_id> <plugin_name>`,
+		Args:              cobra.ExactArgs(2),
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Can you use the RunE function and return errors instead of calling log.Fatal? So it can collect coverage output even from failures